package/MDAnalysis/analysis/hydrogenbonds/hbond_analysis.py

```diff
@@ -219,6 +219,7 @@
 from MDAnalysis.lib.distances import capped_distance, calc_angles
 from MDAnalysis.lib.correlations import autocorrelation, correct_intermittency
 from MDAnalysis.exceptions import NoDataError
+from MDAnalysis import AtomGroup
```

Comment:
I think it would be better to import `AtomGroup` from core.groups (I think...) – or do we normally import the top level alias?

Reply:
> Can the same problem also happen for acceptors or is it restricted to the case of finding H via topology?

The `acceptors` AtomGroup is always created using a selection string (even when the acceptors are guessed with `guess_acceptors`), so this shouldn't be an issue here.

Reply:
woops, I mean to post the above reply at the bottom of the page.

I wasn't sure where to import `AtomGroup`  from, so thanks for pointing it out!